src/netif.c

```diff
@@ -1762,7 +1866,7 @@ static int add_default_gateway(struct netif *nb, const char *ifname, const char
     int idx = ifname_to_index(nb, ifname);
 
     if (!idx) {
-      debug("[%s %d %s]: No such device: '%s'\r\n", __FILE__, __LINE__, __FUNCTION__, ifname);
+      debug("[%s %d]: No such device: '%s'\r\n", __FILE__, __LINE__, ifname);
```

Comment:
why are you removing function name here? Isn't it usually useful for debug logs?

Reply:
ISO was bitching about __FUNCTION__ being not part of the standard :(